sklearn/preprocessing/data.py

```diff
@@ -1917,3 +1920,38 @@ def transform(self, X):
         """
         return _transform_selected(X, self._transform,
                                    self.categorical_features, copy=True)
+
+
+def _boxcox(X):
```

Comment:
Do we need this? Why can't we just do a `stats.boxcox(X)[0]` at the required place(s)?